### DISCUSSION TITLE
tx16s installing Multiprotocols

### DISCUSSION THREAD

**Discussion by rionio** [pascallanger/DIY-Multiprotocol-TX-Module#726] [Q&A]
hello I'm new in this and I need help installing this [protocol](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/master/Multiprotocol/Bayang_Rx_nrf24l01.ino) because I need to use the nrf24l01 receiver with a tx16s radio

**Comment by pascallanger** [comment]
Sorry I don't understand your request...

**Reply by rionio** [reply]
I try to bind the module nrf24l01 which is connected to an arduino with my tx16s and I read that this protocol can do that, but I don't know how to put the protocol in my radio

**Comment by pascallanger** [comment]
The RX protocols are part of the build. Do you just need to select the RX protocols you want just like any other protocol.
